### PR TITLE
refactor FAQ page

### PR DESCRIPTION
**Pull request by GustavoUT22**
I solved many problems
There are design, unnecessary styles, translation and code refactor.

### CODE REVIEW

**Review by Mardecera — CHANGES_REQUESTED** [review]

File: src/components/StillHaveQuestion/StillHaveQuestion.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const StillHaveQuestion: React.FC = () => {
           {intl('frequently.asked.questions.still.have.questions.description')}
         </SupportChat>
       </HaveQuestion>
-      <ButtonLink type="primary" to="/contact-us">
+      <ButtonLink type="primary" to="/#contact-us">
```
[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/components/StillHaveQuestion/StillHaveQuestion.tsx:21]: Done!

File: src/components/StillHaveQuestion/StillHaveQuestion.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ const StillHaveQuestion: React.FC = () => {
...
         Contactanos <TbSend />
```
[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/components/StillHaveQuestion/StillHaveQuestion.tsx:22]: Done!!!
home.navbar.button.contact.us
I used an existing key

File: src/pages/FrequentlyAskedQuestion/FrequentlyAskedQuestion.style.ts [hunk 3]
```diff
@@ -2,30 +2,45 @@ import styled from 'styled-components'
...
-  color: ${colors.gray[50]};
-  font-size: clamp(2rem, 1.667rem + 1.481vw, 3rem);
+  color: ${colors.white.main};
+  font-size: clamp(2rem, 1.667rem + 3.481vw, 4rem);
   text-align: center;
+  font-weight: 700;
+  text-transform: uppercase;
+  line-height: 1.5;
+`
+
```
[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/pages/FrequentlyAskedQuestion/FrequentlyAskedQuestion.style.ts:21]: We have to dicuss that

[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/pages/FrequentlyAskedQuestion/FrequentlyAskedQuestion.style.ts:21]: H1 component has for default do.

File: src/pages/FrequentlyAskedQuestion/FrequentlyAskedQuestion.tsx [hunk 4]
```diff
@@ -5,59 +5,34 @@ import {
...
+    <SectionFAQ id="frequently-asked-questions" size="lg" type="margin">
       <ContainerFaq size="xl">
-        <FaqTitle>{intl('frequently.asked.questions.title')}</FaqTitle>
+        <TitleWrapper>
+          <FaqTitle>{intl('frequently.asked.questions.title')}</FaqTitle>
+        </TitleWrapper>
         <FaqQuestionsWrapper size="sm">
           {questions.map((question) => {
             return (
               <FaqCard
```
[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/pages/FrequentlyAskedQuestion/FrequentlyAskedQuestion.tsx:25]: Done!

[the remark this answers is not in the document]
Comment (mardecera) [reply src/pages/FrequentlyAskedQuestion/FrequentlyAskedQuestion.tsx]: Copy the layout of the multiselect options for the FAQ cards.

Captura de pantalla 2024-08-12 093932

File: src/components/StillHaveQuestion/StillHaveQuestion.tsx [hunk 5]
```diff
@@ -18,8 +18,12 @@ const StillHaveQuestion: React.FC = () => {
           {intl('frequently.asked.questions.still.have.questions.description')}
         </SupportChat>
       </HaveQuestion>
-      <ButtonLink type="primary" to="/contact-us">
-        Contactanos <TbSend />
+      <ButtonLink
+        type="primary"
+        to="/#contact-us"
```
Comment (mardecera) [inline src/components/StillHaveQuestion/StillHaveQuestion.tsx:23]: This should redirect to the Contact Us page, not the section.

**Review by mardecera — CHANGES_REQUESTED** [review]

File: src/pages/FrequentlyAskedQuestion/FrequentlyAskedQuestion.style.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (mardecera) [reply src/pages/FrequentlyAskedQuestion/FrequentlyAskedQuestion.style.ts:21]: Wouldn't it be a better idea to add an optional dot property and make it true by default?

File: src/components/StillHaveQuestion/StillHaveQuestion.tsx (hunk 5 above)
Comment (GustavoUT22) [reply src/components/StillHaveQuestion/StillHaveQuestion.tsx]: Done!

**Review by mardecera — APPROVED** [review]
Nice!

**Review by CrhistianT7 — APPROVED** [review]
LGTM!!!

**Pull request merged**